Libraries/LibGfx/JPGLoader.cpp

```diff
@@ -1112,14 +1112,17 @@ static bool decode_jpg(JPGLoadingContext& context)
 
 static RefPtr<Gfx::Bitmap> load_jpg_impl(const u8* data, size_t data_size)
 {
-    JPGLoadingContext context;
-    context.data = data;
-    context.data_size = data_size;
+    OwnPtr<JPGLoadingContext> context = make<JPGLoadingContext>();
```

Comment:
This is just a hack fix, we need to figure out what's going wrong :)

Reply:
Today's lesson: Always initialize your variables :)

Reply:
Aha there we go! Please initialize all other (primitive) variables as well :)